openmls/src/group/core_group/mod.rs

```diff
@@ -436,8 +436,6 @@ impl CoreGroup {
             let required_capabilities = required_extension.as_required_capabilities_extension()?;
             // Ensure we support all the capabilities.
             required_capabilities.check_support()?;
-            self.own_leaf_node()?
-                .validate_required_capabilities(required_capabilities)?;
```

Comment:
Is there a reason we're not doing validation here anymore?

Reply:
Good point. I re-introduced it (with its new syntax) but commented it out for now. Since the checking is now stricter, we need to wait for #1361 to re-enable this. 

CC @augustocdias